packages/forms-web-app/src/controllers/examination/check-submission-item/utils/get-back-link-url.js

```diff
@@ -0,0 +1,55 @@
+const {
+	getSubmissionItemType,
+	getSubmissionItemPersonalInformation,
+	getActiveSubmissionItemFiles,
+	getActiveSubmissionItem
+} = require('../../session/submission-items-session');
+const {
+	routesConfig: {
+		examination: {
+			directory,
+			pages: {
+				evidenceOrComment,
+				personalInformation,
+				personalInformationComment,
+				personalInformationCommentFiles,
+				personalInformationFiles,
+				personalInformationWhichCommentFiles,
+				personalInformationWhichFiles
+			}
+		}
+	}
+} = require('../../../../routes/config');
+
+const getBackLinkUrl = (session) => {
+	const activeSubmissionItem = getActiveSubmissionItem(session);
+	const submissionItemType = getSubmissionItemType(activeSubmissionItem);
+	const hasPersonalInformation =
+		getSubmissionItemPersonalInformation(activeSubmissionItem) ===
+		personalInformation.options[1].value;
+
+	let backLinkUrl;
+
+	if (submissionItemType === evidenceOrComment.options[1].value) {
```

Comment:
we can try and break this down a bit better. 

Maybe some sub functions for the nested if else. Should make tests simpler.